webuild-drafts/IntermediaryTesting.md

```diff
@@ -0,0 +1,538 @@
+
+## 4.1       Interaction Model
+
+The RPI acts as a complete proxy between the Relying Party and the wallet:
+
+Wallet Holder → EUDIW → RPI → Relying Party
+
+**Characteristics:**
+
+- RPI handles protocol negotiation with wallet
+  
+- RPI performs attestation validation
+  
+- RPI transforms attributes to RP-specific format
+  
+- No direct wallet-to-RP communication
+  
+
+**Trust Implications:**
+
+- RPI processes privacy-sensitive information
+  
+- RP trusts RPI validation results
+  
+- Article 5b 10 prohibits RPIs from storing attribute data
+  
+
+## 4.2       Interaction Flows
+
+### 4.2.1        Presentation Request Flow
+
+1. **RP initiates authentication/attestation request**
+  
+
+- Specifies required attributes and policies
+  
+- Provides session context
+  
+
+2. **RPI generates wallet presentation request**
+  
+
+- Interacts with the user to initiate the EUDIW communication
+  
+- Translates RP requirements to wallet protocol (OpenID4VP, etc.)
+  
+
+3. **Wallet holder accepts EUDIW communication**
+  
+
+- Indicates the EUDIW to use
+  
+- Opens EUDIW and accepts request
+  
+
+4. **Wallet holder authorizes disclosure**
+  
+
+- Reviews requested attributes
+  
+- Consents to disclosure
+  
+- Wallet generates presentation
+  
+
+5. **RPI receives and validates presentation**
+  
+
+- Verifies cryptographic proofs
+  
+- Validates issuer trust status
+  
+- Checks revocation status
+  
+- Checks that the EAA / PID conforms to an EAA Policy that is acceptable for the given use case
+  
+- Applies EAA Policy rules and possibily additional
+  
+
+6. **RPI delivers attributes to RP**
+  
+
+- Formats attributes per RP requirements
+  
+- Provides validation assurance
+  
+- Includes audit information
+  
+
+### 4.2.2        Data Minimization and Privacy
+
+RP intermediaries need to treat a high amount of privacy sensitive information. According to eIDAS 2 they are not allowed to store that data.
+
+**Key Requirements:**
+
+- **No Persistent Storage**: Attribute data must be processed in-memory only
+  
+- **Selective Disclosure**: RPI should request and forward only required attributes
+  
+- **Pseudonymization**: Support for user pseudonyms to limit correlation
+  
+- **Audit Logging**: Maintain non-repudiable audit trails without storing PII
+  
+- **Encryption in Transit**: All communications must use encryption
+  
+
+## 4.3       Trust and Security Requirements
+
+### 4.3.1        Trust Service Provider Status
+
+Draft ETSI TR 119 479-1 Annex B states that RP intermediaries have an important trust role, and as such should be considered similar to a Trust Service Provider. This recognition suggests that RPIs should at least meet security and policy requirements comparable to other TSPs. ETSI EN 319 401 (General Policy Requirements for Trust Service Providers) is a minimal basis. WE BUILD could deliver additional requirements adapted to the specific case of RPIs to ETSI for inclusion in a standard on the topic.
```

Comment:
a statement of a technical report would not be used as a normative reference.

"should considered similar to" would not bring any normative value for the scope of this document

I would suggest more information in the introductionary sections of this document to clearly state the main purpose of this document